test/resources/test-flavor spaces/real-test-flavor/real_flavor_base/build_run/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM {{language_deps}}
+
+COPY --from={{build_deps}} /usr /usr
+RUN true # workaround for https://github.com/moby/moby/issues/37965
+
+COPY --from={{build_deps}} /lib /lib
+RUN true # workaround for https://github.com/moby/moby/issues/37965
+
+COPY --from={{build_deps}} /bin /bin
+RUN true # workaround for https://github.com/moby/moby/issues/37965
+
+COPY --from={{build_deps}} /opt /opt
+RUN true # workaround for https://github.com/moby/moby/issues/37965
+
+COPY --from={{build_deps}} /etc /etc
+RUN true # workaround for https://github.com/moby/moby/issues/37965
```

Comment:
is this a jinja template? if so one could generate this with a loop, passing the path(s) as parameter from within or from the outside of the template, e.g.:
```jinja
{% for path in ['/usr', '/opt', '/lib'] %}
COPY --from={{build_deps}} {{ path}} {{ path }}
RUN true # workaround for https://github.com/moby/moby/issues/37965
{% endfor %}
```